tools/maf_clean_up.pl

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+
+# search for header row
+open(IN, "data_mutations_extended.txt") or die;
+my $headerline = "";
+while (my $line = <IN>) {
+	chomp $line;
+	my @columns = split(/\t/, $line);
+	if ($columns[0] eq "Hugo_Symbol") {
+		$headerline = $line;
+		last;
+	}
+}
+close IN;
+
+# iterate through all column headers and record the index of targeted ones
+my @headers = split(/\t/, $headerline);
+my $count = 0;
+my $exon_number_index = -1;
+my $exon_index = -1;
+my $intron_index = -1;
+my @oncotator_index = ();
+my @keep_cols = ();
+my $header = "";
+foreach $header (@headers) {
+	if (uc($header) eq "EXON_NUMBER") {
+		$exon_number_index = $count;
+	} elsif (uc($header) eq "EXON") {
+		$exon_index = $count;
+	} elsif (uc($header) eq "INTRON") {
+		$intron_index = $count;
+	} elsif (uc(substr($header, 0, length ("ONCOTATOR"))) eq "ONCOTATOR") {
+		push @oncotator_index, $count;
+	} else {
+		push @keep_cols, $count;
+	}
+	$count++; 
+}
+print "INDEX OF COLUMNS TO DROP >>>>>>>>>>>>>>\n";
+print "INTRON: $intron_index\n";
+print "EXON_NUMBER: $exon_number_index\n";
+print "EXON: $exon_index\n";
+print "ONCOTATOR: @oncotator_index\n";
+print "DROPPING COLUMNS ....";
+
+#Reconstruct file
+open(IN, "data_mutations_extended.txt") or die "Error occured while writing MAF...";
+open(DAT, ">out.txt") or die "Error occured while writing .txt output...";
+while (my $line = <IN>) {
+	chomp $line;
+	my @columns = split(/\t/, $line);
+	my $keep_col_index = -1;
+	my $col_counter = 0;
```

Comment:
From what I'm understanding, we're going forward with an assumption Hugo_Symbol is always the first column (so if $col_counter == 0, we apply the date checking). If that is the case, we may end up double printing certain values in the first column (if the value is in the first column and is a corrupted date value).

i.e First column value is 1-Feb
Output will have FEB11-Feb in the first column.